aspnetcore/security/authorization/razor-pages-authorization.md

```diff
@@ -115,6 +115,17 @@ The reverse, however, isn't valid. You can't declare a folder of pages for anony
 
 Requiring authorization on the Private page fails. When both the <xref:Microsoft.AspNetCore.Mvc.Authorization.AllowAnonymousFilter> and <xref:Microsoft.AspNetCore.Mvc.Authorization.AuthorizeFilter> are applied to the page, the <xref:Microsoft.AspNetCore.Mvc.Authorization.AllowAnonymousFilter> takes precedence and controls access.
 
+## Authorize access to anonymous Identity pages
```

Comment:
Delete this entire section. Register uses the `    [AllowAnonymous]` attribute, which is well documented and understood.